### DISCUSSION TITLE
使用有道翻译引擎翻译文档注释 提示    [翻译文档失败: 无相关服务的实例]

### DISCUSSION THREAD

**Discussion by javaNewHard** [YiiGuxing/TranslationPlugin#4312] [Q&A] [answered]
已开通富文本和文档翻译后

尝试过的手段

1. 重启过idea 
2. 切换过翻译引擎 
3. 重新生成并绑定了新的key

![image](https://github.com/YiiGuxing/TranslationPlugin/assets/111348580/e001c1c1-e78a-41c9-9b46-6d83d828ee79)
![image](https://github.com/YiiGuxing/TranslationPlugin/assets/111348580/0ed757d5-c431-4475-bf1c-2418ed6f6fad)

**Comment by YiiGuxing** [comment, marked as answer]
可能是延迟了

**Reply by javaNewHard** [reply]
确实是延迟了 今天可以用了 谢谢您
